src/main/drivers/flash/flash_w25q128fv.c

```diff
@@ -454,8 +462,11 @@ MMFLASH_CODE static uint32_t w25q128fv_pageProgramContinue(flashDevice_t *fdevic
         segments[DATA1].negateCS = false;
         segments[DATA1].callback = NULL;
         segments[DATA2].u.buffers.txData = (uint8_t *)buffers[1];
+#ifdef USE_FLASH_WRITES_USING_4LINES
+        segments[DATA2].len = bufferSizes[1] | BUS_SEGMENT_LEN_WIDTH_X4;
+#else
         segments[DATA2].len = bufferSizes[1];
-        fdevice->bytesWritten += bufferSizes[1];
```

Comment:
this is safe?

Reply:
Ouch! Well spotted. That was an accidental deletion. Thank you!